custom-metrics-stackdriver-adapter/examples/prometheus-to-sd/Dockerfile

```diff
@@ -12,10 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.12.5
-ADD . /go/src/prometheus-dummy-exporter
-RUN go get github.com/prometheus/client_golang/prometheus
+FROM golang:1.16-alpine AS builder
```

Comment:
why "golang:1.16-alpine" not "golang:1.16"?

Reply:
right, `alphine` is a more secure and lightweight version of linux